sailthru_content/services/tests/test_sailthru_api_service.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
+import json
+import mock
+import unittest
+
+from factory import fuzzy
```

Comment:
I believe your comment was meant for @rlucioni. I agree that we should use `faker` if `fuzzy` has been deprecated. See [`fake.pystr()`](https://faker.readthedocs.io/en/latest/providers/faker.providers.python.html).